changelog.d/15515.misc

```diff
@@ -0,0 +1 @@
+Updated Inline SQL queries using boolean parameters as supported in SQLite 3.27.
```

Comment:
I went through the [list that @H-Shay provided](https://github.com/matrix-org/synapse/pull/15525/files#r1202665326) and seems like you're only missing this instance:

 - [`synapse/storage/databases/main/stream.py#L1478`](https://github.com/matrix-org/synapse/blob/e0c39d6bb526b01368393ae5d2173c8e6d39b60f/synapse/storage/databases/main/stream.py#L1478) which [should be `False`](https://github.com/matrix-org/synapse/blob/e0c39d6bb526b01368393ae5d2173c8e6d39b60f/synapse/storage/databases/main/stream.py#L1404)

Reply:
Done.